src/mainviews/job-detail.tsx

```diff
@@ -1,13 +1,398 @@
-import React from 'react';
-
-import { IJobDetailModel, JobsView } from '../model';
+import React, { useEffect, useState } from 'react';
+import { ICreateJobModel, IJobDetailModel, JobsView } from '../model';
+import Box from '@mui/material/Box';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import Accordion from '@mui/material/Accordion';
+import {
+  AccordionDetails,
+  AccordionSummary,
+  Checkbox,
+  CircularProgress,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  OutlinedInputProps,
+  TextField,
+  Typography
+} from '@mui/material';
+import { caretDownIcon } from '@jupyterlab/ui-components';
+import { useTranslator } from '../hooks';
+import { Heading } from '../components/heading';
+import { Scheduler, SchedulerService } from '../handler';
+import { JupyterFrontEnd } from '@jupyterlab/application';
 
 export interface IJobDetailProps {
+  app: JupyterFrontEnd;
   model: IJobDetailModel;
   modelChanged: (model: IJobDetailModel) => void;
```

Comment:
Nothing for you to do right now, but when #44 is merged in, there will be a conflict because this is being renamed to `handleModelChange`.